### ISSUE TITLE
Wait command should not be accepted from dev who is not assigned to the issue

### ISSUE THREAD

**Issue by pinaf** [zerocracy/farm#2081]
Currently if a project dev asks 0crat to wait on an issue that command is accepted even when the dev is no longer the one assigned to the issue. Let's make 0crat refuse these requests unless they come from the assigned dev or from the ARC or the PO.

![image](https://user-images.githubusercontent.com/1054771/59526195-b7c36380-8eae-11e9-8545-040d2ffb8b20.png)

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@pinaf[/z](https://www.0crat.com/u/pinaf) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by 0crat** [comment]
Job [#2081](https://github.com/zerocracy/farm/issues/2081) is now in scope, role is `DEV`

**Comment by 0crat** [comment]
Job `gh:zerocracy/farm#2081` already assigned to @g4s8, can't assign to @g4s8

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) not enough funds available in the project, can't set budget of job [#2081](https://github.com/zerocracy/farm/issues/2081), see [§21](http://www.zerocracy.com/policy.html#21); @g4s8[/z](https://www.0crat.com/u/g4s8) will get no money on completion; in order to fix that, add funds to the project and assign the job again

**Comment by 0crat** [comment]
Bug was reported, see [§29](http://www.zerocracy.com/policy.html#29): +15 point(s) just awarded to @pinaf[/z](https://www.0crat.com/u/pinaf)

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by g4s8** [comment]
@pinaf it should be fixed in #2083 and released soon.

**Comment by pinaf** [comment]
@g4s8 thank you

**Comment by 0crat** [comment]
Job [#2081](https://github.com/zerocracy/farm/issues/2081) is not in the agenda of @g4s8[/z](https://www.0crat.com/u/g4s8), can't retrieve data and time of add

**Comment by 0crat** [comment]
Job was finished in 6 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
The job [#2081](https://github.com/zerocracy/farm/issues/2081) is now out of scope

**Comment by 0crat** [comment]
Order was finished: +35 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)
